fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/security/Base64URLData.java

```diff
@@ -37,6 +35,10 @@ private static byte[] decode(String data) {
     return Base64.getUrlDecoder().decode(data);
   }
 
+  public static String encode(String data) {
+    return encode(data.getBytes());
```

Comment:
does `getBytes()` assume a particular encoding, e.g. UTF-8 ?

Reply:
It uses the default charset of the platform, which is usually UTF-8, but to be sure I'll force the charset specifying it as param